### DISCUSSION TITLE
Setting IP in 1.13.2 tasmota32

### DISCUSSION THREAD

**Discussion by msetina** [arendst/Tasmota#19821] [Q&A]
Is there a change in new firmware for setting WIFI? I experience IPAddress1 not working in Backlog command with settings WIFI AP SSID and password after flashing firmware. After reboot the same Backlog command executed in Console WEB works perfectly.
Should this be in separate Backlog commands.
In previous versions it worked.
It gets a DHCP IP.

**Comment by msetina** [comment]
This does not impact tasmota build.

**Reply by sfromis** [reply]
What do you see in the log when issuing the commands not getting the expected outcome?

**Reply by msetina** [reply]
For both tasmota and tasmota32 I have the same process:
1. flash with esptool (after that procedure wait 10s for reset)
2. set wifi with fixed IP over serial with mentioned BAcklog command (Backlog SSID1 <SSID>; Password1 <pwd>; IPAddress1 <fixed IP>; IPAddress2 <gateway>; IPAddress3 <netmask>; IPAddress4 <DNS>;)
3. set template
....
in previous version of tasmota32 and on current version of tasmota after the procedure fixed IP was set after reboot.
I checked after reboot running the same Backlog command and the fixed IP is set.
The Backlog command does not return anything when I ran from console, so the procedure also does not get anything back.
The expected result is for the fixed IP to be set, but in the specific case it is not. I am ready to change the procedure to first set the SSID information for IP and then set the fixed IP, but I am afraid that inbetween DHCP is invoked.

Since multiple resets are made I do not know how to get log information since it is right after flashing.

**Reply by msetina** [reply]
I removed values from Backlog command.....

**Reply by sfromis** [reply]
Tasmota will always respond to a command, including in the console, unless you specifically reduced log level below the default of 2.

**Reply by msetina** [reply]
I took image tasmota32.factory.bin from release repository and flashed with esptool. I never touched log level.

**Reply by sfromis** [reply]
Then it is incorrect to not expect responses every time you issue a command. In fact, not getting a response would indicate a problem.
